src/common/collection.c

```diff
@@ -470,57 +555,115 @@ gboolean dt_collection_get_sort_descending(const dt_collection_t *collection)
 gchar *dt_collection_get_sort_query(const dt_collection_t *collection)
 {
   gchar *sq = NULL;
+  gchar *second_order = NULL;/*remember previous sorting criteria as second order sorting criteria*/
+
+  switch(collection->params.sortSecondOrder)/*build ORDER BY string for second order*/
+  {
+        case DT_COLLECTION_SORT_DATETIME:
```

Comment:
I feel that the indentation is far too much here. Can you double check?

Reply:
True, done.